pysat/_meta.py

```diff
@@ -494,8 +496,7 @@ def __getitem__(self, key):
         """
         # Define a local convenience function
         def match_name(func, var_name, index_or_column):
-            """Applies func on input variables(s) depending on variable type
-            """
+            """Apply func on input variables(s) depending on variable type."""
```

Comment:
```suggestion
            """Alter variables using input function."""
```

May or may not be a better description.